frontend/javascripts/admin/time/time_line_view.js

```diff
@@ -201,7 +202,13 @@ class TimeLineView extends React.PureComponent<*, State> {
               chartType="Timeline"
               columns={columns}
               rows={rows}
-              options={{ timeline: { singleColor: "#108ee9" } }}
+              options={{
+                timeline: { singleColor: "#108ee9" },
+                hAxis: {
```

Comment:
Maybe add a comment here with a reference to this PR? Otherwise, we might forget about it and re-introduce the bug again at some point :see_no_evil: